### ISSUE TITLE
vist_train_image_features.hdf5

### ISSUE THREAD

**Issue by 22881916** [wasiahmad/visual_story_telling#1]
Hello Wasiahmad,
I have a question about the vist_train_image_features.hdf5.
I have downloaded the VIST dataset (300G). But there is no vist_train_image_features.hdf5.
Is the file you made by yourself? Could you tell me more about this?
Thank you.

**Comment by wasiahmad** [comment]
Yes, that file is generated by me and the file contains image features generated using VGGNet.

**Comment by 22881916** [comment]
Thanks a lot.
Could you please tell me more about the details? 
Or send me the two files(vist_train_image_features.hdf5 and vist_dev_image_features.hdf5) or download link?
My email is ray.wangrz@outlook.com.
